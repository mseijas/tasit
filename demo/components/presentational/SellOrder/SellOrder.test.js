import React from "react";
import { shallow } from "enzyme";
import SellOrder from "./SellOrder";

describe("SellOrder", () => {
  it("renders the component", async () => {
    const sellOrder = {
      id: -1,
      priceMana: 0,
      priceUSD: 0,
      asset: {
        id: -1,
        name: "Not found",
        img: "https://decentraland.org/images/logo-65f7b27caf.png",
      },
    };
    expect(shallow(<SellOrder sellOrder={sellOrder} />)).toMatchSnapshot();
  });
});
