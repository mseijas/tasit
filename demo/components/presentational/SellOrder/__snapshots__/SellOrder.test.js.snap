// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrder renders the component 1`] = `
<Component
  style={
    Object {
      "width": 450,
    }
  }
>
  <Estate
    estate={
      Object {
        "id": -1,
        "img": "https://decentraland.org/images/logo-65f7b27caf.png",
        "name": "Not found",
      }
    }
  />
  <Component>
    0
     MANA
  </Component>
</Component>
`;
